src/Tasks/Microsoft.NET.Build.Tasks/ProduceContentAssets.cs

```diff
@@ -228,21 +228,22 @@ private void ProduceContentAsset(ITaskItem contentFile)
             string pathToFinalAsset = resolvedPath;
             string ppOutputPath = contentFile.GetMetadata(PPOutputPathKey);
             string parentPackage = contentFile.GetMetadata(MetadataKeys.ParentPackage);
+            string[] parts = parentPackage?.Split('/');
+            if (parts == null || parts.Length != 2)
+            {
+                throw new BuildErrorException(Strings.ContentFileDoesNotContainExpectedParentPackageInformation, contentFile.ItemSpec);
+            }
+            string packageName = parts[0], packageVersion = parts[1];
```

Comment:
(nit) I think it is uncommon to make assignments like this in our current code base.  Can these be split into 2 lines?